bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/votable/VoTableClient.scala

```diff
@@ -173,8 +178,44 @@ case object RemoteBackend extends CachedBackend {
 
 }
 
+case object SimbadNameBackend extends CachedBackend {
+  val instance = this
+  override val catalogUrls = NonEmptyList(new URL("http://simbad.cfa.harvard.edu/simbad/"), new URL("http://simbad.u-strasbg.fr/simbad"))
+
+  val Log = Logger.getLogger(getClass.getName)
+
+  private val timeout = 30 * 1000 // Max time to wait
+
+  protected [votable] def queryParams(q: CatalogQuery): Array[NameValuePair] = q match {
+    case qs: NameCatalogQuery => Array(
+      new NameValuePair("output.format", "VOTable"),
+      new NameValuePair("Ident", qs.search))
+    case _                    => Array.empty
+  }
+
+  override protected def query(e: SearchKey): QueryResult = {
+    val method = new GetMethod(s"${e.url}/sim-id")
+    val qs = queryParams(e.query)
+    method.setQueryString(qs)
+    Log.info(s"Catalog query to ${method.getURI}")
+
+    val client = new HttpClient
+    client.setConnectionTimeout(timeout)
+
+    try {
+      client.executeMethod(method)
+      VoTableParser.parse(e.url, method.getResponseBodyAsStream).fold(p => QueryResult(e.query, CatalogQueryResult(TargetsTable.Zero, List(p))), y => QueryResult(e.query, CatalogQueryResult(y)))
```

Comment:
For things like this I prefer a match to a fold because it allows you to compare the cases visually (you could also do this by adding newlines but I don't really like newlines inside parens). Not a huge deal but I think this reads better.

```scala
VoTableParser.parse(e.url, method.getResponseBodyAsStream) match {
  case -\/(p) => QueryResult(e.query, CatalogQueryResult(TargetsTable.Zero, List(p)))
  case \/-(y) => QueryResult(e.query, CatalogQueryResult(y)))
}
```